src/native/NativeStyleTransfer.ts

```diff
@@ -0,0 +1,10 @@
+import type { TurboModule } from 'react-native';
+import { TurboModuleRegistry } from 'react-native';
+
+export interface Spec extends TurboModule {
+  loadModule(modelSource: string): Promise<number>;
+
+  forward(input: string): Promise<string>;
```

Comment:
Is string input the only possible option? Maybe we should allow passing plain arrays as well

Reply:
I believe that we talked about it and decided to let users pass plain arrays with bindings and for dedicated hooks it should accept more "high level" input so in this case url or base64.

Reply:
ah, i thought we were doing both, but we can stick to strings then